internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -45,13 +45,26 @@ username: %s
 password: %s
 `
 
+const quickstartTemplateIntro = `You are creating a new Atlas cluster and enabling access to it.
+
+Press [Enter] to use the default values.
```

Comment:
[q] Should we show the defaults that will be used to the user at this point - like a summary of what's going to happen.

Reply:
We are showing the default value between parenthesis for each question.
`? Cluster Name [This can't be changed later] (Quickstart-1625578411)`

I think that if we add all the default values at the beginning we may force the user to scroll up all the time to check what is the default value for a particular question.

Reply:
Fair point, sgtm